stun_server/region/module.tf

```diff
@@ -0,0 +1,37 @@
+locals {
+  service_name = "stun-server"
+  launch_type  = "FARGATE"
+}
+
+data "tfe_outputs" "infrastructure" {
+  organization = "home_assistant"
+  workspace    = "infrastructure"
+}
+
+module "stun_server" {
+  source = "../../.modules/service"
+
+  service_name      = local.service_name
+  container_image   = "ghcr.io/home-assistant/stun-server"
```

Comment:
https://github.com/home-assistant/core/pkgs/container/stun-server returns 404

Reply:
I made a release and `ghcr.io/home-assistant/stun:1.0.0` returns an image now. Your provided URL still returns 404, but I don't think it is ever used in this context.

Reply:
As the requested image here is stun-server and not stun, yes it is used in this context.

Reply:
🤡